static/HR_density.json

```diff
@@ -0,0 +1,16637 @@
+[
```

Comment:
This is really nice!

Reply:
Thanks! also wanted to ask: is there a trick to checking the xpath for a vega plot? I added `data-testid` but I don't see it anywhere on the inspect element and it doesn't find it when running the test...